.github/workflows/snyk.yml

```diff
@@ -0,0 +1,27 @@
+name: Snyk dependencies check
```

Comment:
Is the Snyk check "free"?  If so, nice addition.

Reply:
It's free in the sense that number of tests/checks are unlimited when you pay for a license. And even if we didn't it, would be free because this repo is open source.